### PR TITLE
Packaging clef option

### PR DESCRIPTION
**Pull request by vandot**
Create (hopefully) better support for enabling clef during bee install.

### CODE REVIEW

File: packaging/deb/postinst [hunk 1]
```diff
@@ -14,11 +15,35 @@ if [ "$1" = "configure" ]; then
...
+        case "$RESP" in
+            Error*)
+                START=false
+                echo "Enabled clef-signer but clef is not running."
+                echo "Check https://docs.ethswarm.org/ for more info and how to fix."
+                echo "Or install latest release of bee-clef from https://github.com/ethersphere/bee-clef and reinstall bee."
+                echo "Start bee with systemctl --no-reload start bee.service"
+                ;;
+            *)
+                ETH_ADDRESS=$(echo "$RESP" | grep ethereum | cut -d' ' -f6 | tr -d '"')
```
Comment (janos) [inline packaging/deb/postinst:38]: We should make output of init command standarized for parsing with an optional flag instead to relay on cut fields of the log message. Someone may change the log message and break this requirement.

File: packaging/rpm/post [hunk 2]
```diff
@@ -1,23 +1,33 @@
...
         parse_json() { echo $1|sed -e 's/[{}]/''/g'|sed -e 's/", "/'\",\"'/g'|sed -e 's/" ,"/'\",\"'/g'|sed -e 's/" , "/'\",\"'/g'|sed -e 's/","/'\"---SEPERATOR---\"'/g'|awk -F=':' -v RS='---SEPERATOR---' "\$1~/\"$2\"/ {print}"|sed -e "s/\"$2\"://"|tr -d "\n\t"|sed -e 's/\\"/"/g'|sed -e 's/\\\\/\\/g'|sed -e 's/^[ \t]*//g'|sed -e 's/^"//' -e 's/"$//' ; }
+        echo "If you don't want to use external signer even if we recommended it."
         echo "Please make sure there is sufficient eth and bzz available on $(parse_json $(cat /var/lib/bee/keys/swarm.key) address)"
         echo "You can get both goerli eth and goerli bzz from https://faucet.ethswarm.org"
     fi
     systemctl --no-reload preset bee.service &>/dev/null || :
     systemctl --no-reload enable bee.service &>/dev/null || :
-    echo "Service already enabled, after initial configuration"
+    echo "If you enabled external signer. Run"
+    echo "/usr/bin/bee init --config /etc/bee/bee.yaml 2>&1 | grep ethereum | cut -d' ' -f6 | tr -d '"'"
```
Comment (janos) [inline packaging/rpm/post:21]: For here, also.

Comment (vandot) [reply packaging/rpm/post:21]: I agree, but being really short on time :man_shrugging:
I think @Eknir already created a ticket for that

Comment (janos) [reply packaging/rpm/post:21]: Maybe to add a comment where the log function is called to state that it is required to be compatible with this parsing?

Comment (janos) [reply packaging/rpm/post:21]: @vandot could you add a comment in cmd/bee/start.go:311 that this log message is required in such form for purposes defined here? I think that this is good to have there for other developers.

**Review by janos — APPROVED** [review]
Thanks @vandot!

**Review by acud — APPROVED** [review]

**Pull request merged**